contracts/interchain-token-service/src/test.rs

```diff
@@ -9,59 +9,38 @@ use soroban_sdk::testutils::{MockAuth, MockAuthInvoke};
 
 use soroban_sdk::{testutils::Address as _, Address, Env, String, Symbol};
 
-fn setup_env<'a>() -> (Env, Address, InterchainTokenServiceClient<'a>) {
+fn setup_env<'a>() -> (Env, InterchainTokenServiceClient<'a>) {
     let env = Env::default();
-
-    let contract_id = env.register_contract(None, InterchainTokenService);
+    let owner = Address::generate(&env);
+    let contract_id = env.register(InterchainTokenService, (&owner,));
     let client = InterchainTokenServiceClient::new(&env, &contract_id);
 
-    (env, contract_id, client)
-}
-
-fn initialize(_env: &Env, client: &InterchainTokenServiceClient, owner: Address) {
-    client.initialize_its(&owner);
+    (env, client)
 }
 
 #[test]
 fn initialize_succeeds() {
```

Comment:
```suggestion
fn register_interchain_token_service() {
```
alternatively `register_contract_succeeds`